packages/editor/src/plugins/image/controls/upload-button.tsx

```diff
@@ -14,14 +13,10 @@ export function UploadButton({ config, state }: ImageProps) {
 
   return (
     <>
-      <label
-        className={cn(`
-          mr-2 cursor-pointer rounded-md border border-gray-500
-          px-1 text-sm transition-all focus-within:bg-editor-primary-200 hover:bg-editor-primary-200
-          focus-visible:bg-editor-primary-200
-        `)}
-      >
-        {imageStrings.upload} <FaIcon icon={faCircleArrowUp} />
+      <label className="mb-4 flex w-full items-center justify-center rounded-lg bg-editor-primary-200 py-2 font-semibold text-gray-800 hover:bg-editor-primary-300">
```

Comment:
This button doesn't have any styling indication when it's been focused using the Tab key.

Reply:
Done